infra/capi/nephio-workload-cluster/pv-crds.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: PackageVariant
+metadata:
+  name: example-crds
+spec:
+  annotations:
+    approval.nephio.org/policy: initial
+  upstream:
+    package: nephio-workload-crds
+    repo: nephio-workload-crds
```

Comment:
```suggestion
    package: nephio/core/workload-crds
    repo: catalog
```

Reply:
packages cannot be in subdirectories, it is the K8s name not the directory. That's why we have to register the subdirectories as repos themselves.

Reply:
So, we will have a repo called "nephio-core" which maps to the nephio/core directory, then the package name could be "workloads-crds" in there.

Reply:
I talked with @electrocucaracha about this, I tried the description so my reference in PV should be fine. 

```
ubuntu@nephio-r1-e2e:~$ kubectl get repositories
NAME                      TYPE   CONTENT   DEPLOYMENT   READY   ADDRESS
catalog                   git    Package   false        True    https://github.com/nephio-project/catalog.git
free5gc-packages          git    Package   false        True    https://github.com/nephio-project/free5gc-packages.git
mgmt                      git    Package   true         True    http://172.18.0.200:3000/nephio/mgmt.git
mgmt-staging              git    Package   false        True    http://172.18.0.200:3000/nephio/mgmt-staging.git
nephio-example-packages   git    Package   false        True    https://github.com/arora-sagar/nephio-example-packages.git
nephio-workload-crds      git    Package   false        True    https://github.com/nephio-project/catalog.git
oai-packages              git    Package   false        True    https://github.com/OPENAIRINTERFACE/oai-packages.git
```